packages/boot/test/bootstrapTests/ec-membership-update.test.ts

```diff
@@ -0,0 +1,399 @@
+    runUtils,
+    storage,
+    agoricNamesRemotes,
+  );
+
+  const { fromCapData } = makeMarshal(undefined, slotToBoardRemote);
+
+  const getUpdatedDebtLimit = () => {
+    const atomGovernance = unmarshalFromVstorage(
+      storage.data,
+      'published.vaultFactory.managers.manager0.governance',
+      fromCapData,
+      -1,
+    );
+    return atomGovernance.current.DebtLimit.value.value;
+  };
+
+  const governanceDriver = await makeGovernanceDriver(
+    swingsetTestKit,
+    agoricNamesRemotes,
+    walletFactoryDriver,
+    wallets,
+  );
+
+  return {
+    ...swingsetTestKit,
+    storage,
+    getUpdatedDebtLimit,
+    governanceDriver,
+  };
+};
+const test = anyTest as TestFn<Awaited<ReturnType<typeof makeZoeTestContext>>>;
+
+test.before(async t => {
+  t.context = await makeZoeTestContext(t);
+});
+
+test.serial('normal running of committee', async t => {
+  const { advanceTimeBy, storage, getUpdatedDebtLimit, governanceDriver } =
+    t.context;
+
+  const agoricNamesRemotes = makeAgoricNamesRemotesFromFakeStorage(storage);
+  const { VaultFactory, economicCommittee } = agoricNamesRemotes.instance;
+  const { ATOM: collateralBrand, IST: debtBrand } = agoricNamesRemotes.brand;
+
+  const committee = governanceDriver.ecMembers;
+
+  t.log('Accepting all invitations for original committee');
+  await null;
+  for (const member of committee) {
+    await member.acceptCharterInvitation(offerIds.propose.outgoing);
+    await member.acceptCommitteeInvitation(offerIds.vote.outgoing);
+  }
+
+  t.log('Proposing a question using first wallet');
+  await governanceDriver.proposeParams(
+    VaultFactory,
+    { DebtLimit: { brand: debtBrand, value: 100_000_000n } },
+    { paramPath: { key: { collateralBrand } } },
+    committee[0],
+    getQuestionId(1),
+    offerIds.propose.outgoing,
+  );
+
+  t.log('Checking if question proposal passed');
+  t.like(committee[0].getLatestUpdateRecord(), {
+    status: { id: getQuestionId(1), numWantsSatisfied: 1 },
+  });
+
+  t.log('Voting on question using first 3 wallets');
+  await governanceDriver.enactLatestProposal(
+    committee,
+    getVoteId(1),
+    offerIds.vote.outgoing,
+  );
+
+  t.log('Checking if votes passed');
+  for (const w of committee.slice(0, 3)) {
+    t.like(w.getLatestUpdateRecord(), {
+      status: { id: getVoteId(1), numWantsSatisfied: 1 },
+    });
+  }
+
+  t.log('Waiting for period to end');
+  await advanceTimeBy(1, 'minutes');
+
+  t.log('Verifying outcome');
+
+  const lastOutcome = await governanceDriver.getLatestOutcome();
+  console.log(lastOutcome);
+  t.deepEqual(getUpdatedDebtLimit(), 100_000_000n);
+  t.assert(lastOutcome.outcome === 'win');
+});
+
+test.serial(
+  'check high priority senders before replacing committee',
+  async t => {
+    const { storage } = t.context;
+
+    const data: any = storage.toStorage({
+      method: 'children',
+      args: [highPrioritySenderKey],
+    });
+    t.deepEqual(data.sort(), wallets.slice(0).sort());
```

Comment:
The `.slice(0)` is to make a copy?

Not very important, but we tend to use...

```suggestion
    t.deepEqual(data.sort(), [...wallets].sort());
```

Reply:
updated